helm/charts/determined/templates/NOTES.txt

```diff
@@ -26,6 +26,12 @@ ERROR: Secret {{ .Values.tlsSecret }} does not exist in namespace: {{ .Release.N
   {{ end }}
 {{ end -}}
 
+{{- if (contains "dev" .Chart.AppVersion) }}
```

Comment:
non-blocking: Should this also exclude things like `rc` versions? If so, it could be a bit tighter using a condition like `not (regexMatch "[0-9]+\\.[0-9]+\\.[0-9]+" .Chart.AppVersion)` (untested) to whitelist instead.

Reply:
Hopefully the updated error message clears this up, but `rc` candidates are fine to install since we publish docker images for them.